packages/react/src/file-upload/utils.test.tsx

```diff
@@ -1,8 +1,13 @@
+import { FileWithPath } from 'react-dropzone';
 import {
 	getErrorSummary,
 	getAcceptedFilesSummary,
 	formatFileSize,
 	formatFileExtension,
+	sortFileByName,
```

Comment:
Dan, I noticed you like to alphabetise imports. This isn't necessarily a request for a change, I just wanted to flag that these ones aren't alphabetised, in case it's something you missed and wanted to come back to. Up to you.

Reply:
That is because I use the `organize imports` utility in VS code to clean up import statements. It alphabetically orders the imports as part of the clean up. (`ctrl+shift+P` -> "organize imports")

I've ran it on utils.test.tsx now.

Reply:
Ahh TIL! Nice one, Dan 🙌